authority-portal-frontend/src/app/core/api/fake-backend/impl/fake-connectors.ts

```diff
@@ -353,6 +354,41 @@ export const createProvidedConnector = (
   };
 };
 
+export const createProvidedConnectorWithJwks = (
+  request: CreateConnectorWithJwksRequest,
+  clientOrganizationId: string,
+): CreateConnectorResponse => {
+  const hostOrganizationId = getUserInfo().organizationId;
+  const hostOrgName = getUserInfo().organizationName;
+  const status = 'OFFLINE';
+
+  const clientOrgName = TEST_ORGANIZATIONS.filter(
+    (it) => it.id === clientOrganizationId,
+  )[0].name;
```

Comment:
`filter(...)[0]` is the same as `find(...)`

Edit: also fix where u copied it from !